Comp2023/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -233,8 +247,12 @@ private void initDefaultCommands( )
    */
   private void initAutonomousChooser( )
   {
+    // Autonomous Chooser
```

Comment:
If your testing seems pretty done on the test paths, the autonomous chooser can be set up with our known autonomous routines. AutoStop is there, but how about AutoDriveOffCommunity, AutoDeliverPreloadAndDrive, AutoDockOnChargeStation.

Reply:
I can add that in right now!